### PR TITLE
zs-assignment-java-1

### PR DESCRIPTION
**Pull request by Deepjyoti13**
Added the code along with .gitignore

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'java'
+}
+
+group 'org.example'
```
Comment (ujjawalmisra) [inline build.gradle:5]: group in build.gradle is supposed to represent which artifact package hierarchy does this project belong to. Most of the times its the base package of the project. Please correct.

File: src/main/java/assignment_1/SystemInfo.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import java.io.File;
+import java.util.Properties;
+
+public class SystemInfo {
+    private static final String osName = System.getProperty("os.name");
+    private static final String osVersion = System.getProperty("os.version");
+    private static final String userName = System.getProperty("user.name");
+    private static final String userHome = System.getProperty("user.home");
+    private static final String jvmHome = System.getProperty("java.home");
+    private static final Long memory = Runtime.getRuntime().totalMemory();
```
Comment (ujjawalmisra) [inline src/main/java/assignment_1/SystemInfo.java:12]: Does this represent the System memory of the memory of the Process of your application? The ask was to find the memory in the system.

**Review by ujjawalmisra — CHANGES_REQUESTED** [review]
You also need to figure out the CPUs in the system.

File: src/main/java/assignment_1/SystemInfo.java (hunk 2 above)
Comment (Deepjyoti13) [reply src/main/java/assignment_1/SystemInfo.java:12]: Sir, this outputs the memory allotted in the JVM.

**Pull request closed without merging**